src/cloud-api-adaptor/pkg/securecomms/sshproxy/sshproxy.go

```diff
@@ -154,6 +116,43 @@ func (inbounds *Inbounds) DelAll() {
 	inbounds.list = [](*Inbound){}
 }
 
+func ParseTag(tag string) (port int, host, name, phase string, err error) {
```

Comment:
I'm not familiar enough with that code to understand what a "tag" is in this context. What do you tag?

Would it be possible to add that information to the cover letter and to a comment above the function?

Reply:
The Inbounds and Outbounds tunnel services are defined and configured to the system using a set of strings (or tags as they are being referred to). As an example, an outbound tag such as `K:ABC:myhost.com:1234` means  that during the `Kubernetes phase`, an output of a tunnel named `ABC` is registered, connection connecting to this tunnel will be forwarded to `myhost.com` port `1234`). 

I will update the docs/SecureComms doc to be more specific about this.